src/api/include/migraphx/migraphx.h

```diff
@@ -197,6 +203,32 @@ migraphx_status migraphx_parse_onnx_buffer(migraphx_program_t* out,
                                            size_t size,
                                            migraphx_onnx_options_t options);
 
+migraphx_status migraphx_quantize_op_names_destroy(migraphx_quantize_op_names_t quantize_op_names);
+
+migraphx_status migraphx_quantize_op_names_create(migraphx_quantize_op_names_t* quantize_op_names);
+
+migraphx_status migraphx_quantize_op_names_add(migraphx_quantize_op_names_t quantize_op_names,
+                                               const char* name);
+
+migraphx_status migraphx_quantize_fp16(migraphx_program_t prog, migraphx_quantize_op_names_t name);
+
+migraphx_status migraphx_quantize_fp16_default(migraphx_program_t prog);
```

Comment:
Can we shorten this to `migraphx_quantize_fp16` and call the other overload `migraphx_quantize_fp16_with_names`.

Reply:
fixed